recipe/patches/0006-Support-cross-compiling-byte-code.patch

```diff
@@ -1,28 +1,28 @@
-From f60350660259e9f0fbf666858ba95c64989e9361 Mon Sep 17 00:00:00 2001
+From 6e92fc0ff5d7a53f425abd9dd759dd7fd0fa0ce0 Mon Sep 17 00:00:00 2001
 From: Ray Donnelly <mingw.android@gmail.com>
 Date: Wed, 16 Aug 2017 12:17:33 +0100
-Subject: [PATCH 07/20] Support cross-compiling byte-code
+Subject: [PATCH 06/18] Support cross-compiling byte-code
 
```

Comment:
When I was trying to rebase the patches for #261, I noticed that this patch has a bunch of calls to `$(PYTHON_FOR_BUILD)` that are not prefixed by `$(PY_BUILD_ENVIRON)`.  I'm guessing those cases might have been introduced upstream without the patch ever being updated, so I thought `$(PY_BUILD_ENVIRON)` should be added to all callsites...?